tests/rptest/services/redpanda.py

```diff
@@ -1330,6 +1330,24 @@ def raise_on_bad_logs(self, allow_list=None):
         if bad_lines:
             raise BadLogLines(bad_lines)
 
+    def search_log(self, pattern):
```

Comment:
> rptest: Move method under redpanda service

this commit doesn't seem to be moving anything (in that it implies there would be deletions) ?